tests/accounts/test_serializers.py

```diff
@@ -25,7 +38,22 @@ def setUp(self):
         self.user.student.school = "School"
         self.serializer = StudentSerializer(self.user.student)
 
-    def test_str(self):
+        self.user_preferred_name = get_user_model().objects.create_user(
```

Comment:
Note to self: We need to delete the org and services apps, and then refactor the StudentSerializer to just list out major, school and year.